production/helm/loki/templates/monitoring/_helpers-monitoring.tpl

```diff
@@ -11,7 +11,19 @@ Client definition for LogsInstance
 {{- end }}
 - url: {{ $url }}
   externalLabels:
-    cluster: {{ include "loki.fullname" . -}}
+    cluster: {{ include "loki.fullname" . }}
+  {{- if or .Values.loki.auth_enabled .Values.enterprise.enabled }}
+  tenantId: {{ .Values.monitoring.selfMonitoring.tenant }}
```

Comment:
is this identation right?

Reply:
Why is a tenant ID required for enterprise?

Reply:
probably not required for enterprise since with enterprise we're providing the username below.

Reply:
changed the logic a bit, and yes double checked indenation, following with with `auth_enabled = true` but enterprise off:
```
spec:
  clients:
    
    - url: http://loki-gateway.default.svc.cluster.local/loki/api/v1/push
      externalLabels:
        cluster: release-name-loki
      tenantId: self-monitoring
```